server/listen.c

```diff
@@ -1038,6 +1043,10 @@ static const char *parse_listen_flags(apr_pool_t *temp_pool, const char *arg,
             flags |= AP_LISTEN_REUSEPORT;
         else if (ap_cstr_casecmp(token, "v6only") == 0)
             flags |= AP_LISTEN_V6ONLY;
+#ifdef IPPROTO_MPTCP
+        else if (ap_cstr_casecmp(token, "multipathTCP") == 0)
+            flags |= AP_LISTEN_MPTCP;
+#endif
```

Comment:
```suggestion
        else if (ap_cstr_casecmp(token, "multipathTCP") == 0)
#ifdef IPPROTO_MPTCP
            flags |= AP_LISTEN_MPTCP;
#else
            return apr_psprintf(temp_pool, "Listen option '%s' in '%s' is not supported on this system",
                                token, arg);
#endif
```

Just not to ignore it.